PlatformIO/src/esp_types.h

```diff
@@ -0,0 +1,7 @@
+
+
+#define ESP_OTHER 0
+#define ESP32_POE_ISO 1
+
+
+#define ESP_TYPE ESP32_POE_ISO
```

Comment:
Please remove this define, because it now defaults to ESP32_POE_ISO.
Add it to cpp_defines in load_settings.py
Probably add a setting under general in setting.ini.example or add an extra section for the device.

Reply:
I'm not very familiar with the syntax of the settings.ini.example. How would I add a new section there. Well at what places do I also have to make changes I mean?

Reply:
Ah! I see